src/plugins/dam/sagas/uploadFileFlow.js

```diff
@@ -28,51 +28,37 @@ export default function* (hashName, fileInfo, fileBuffer, preSignedUrl, variant)
     return true;
   }
 
+  let uploadSucceeded = false;
   for (let retries = 0; retries < MAX_RETRIES; retries += 1) {
     try {
-      // Delay the process by 2 seconds if this isn't the first try
-      if (retries) {
-        yield delay(2000);
-      }
+      yield delay(retries * 2000);
```

Comment:
why are the 2 second delays being multiplied?

Reply:
@rsumilang simple way to have no delay at first and then an increasing delay after failures. is your concern that a total failure will happen after too long of a delay?

Reply:
@efjacobson I think further increasing delay times is unnecessary. Increasing the MAX_RETRIES should be sufficient.

Reply:
@rsumilang updated